programs/voter-stake-registry/src/instructions/close_voter.rs

```diff
@@ -13,17 +16,48 @@ pub struct CloseVoter<'info> {
         has_one = voter_authority,
         close = sol_destination)]
     pub voter: AccountLoader<'info, Voter>,
+
+    #[account(mut)]
     pub voter_authority: Signer<'info>,
+
+    #[account(mut)]
     pub sol_destination: UncheckedAccount<'info>,
+
+    pub token_program: Program<'info, Token>,
 }
 
-/// Closes the voter account, allowing one to retrieve rent exemption SOL.
+/// Closes the voter account (Optionally, also token vaults, as part of remaining_accounts),
+/// allowing one to retrieve rent exemption SOL.
 /// Only accounts with no remaining deposits can be closed.
```

Comment:
Please add documentation for the meaning and flags needed (write I assume) on the remaining_accounts at the end of the account struct (see configure_voting_mint)